pkg/xds/envoy/listeners/v3/retry_configurer.go

```diff
@@ -93,6 +93,21 @@ func genHttpRetryPolicy(
 		policy.RetriableStatusCodes = conf.RetriableStatusCodes
 	}
 
+	for _, methodValue := range conf.GetRetriableMethods() {
```

Comment:
Since you can drop the local variable `method`, you can call this one either `method` or just `m` since the loop is short.